scripts/build-shaders.js

```diff
@@ -33,12 +33,15 @@ var data = data.split('\n').map(function(line) {
         } else {
             return params[1] + params[3] + 'p ' + params[4] + ' ' + params[5] + ' = u_' + params[5] + ';';
         }
+    } else if (line.match(pixel_ratio_regex)){
+        return '\n#define DEVICE_PIXEL_RATIO 2.0\n' + line;
```

Comment:
This line replaces any occurrences of `/(.*)#define(.*)DEVICE_PIXEL_RATIO(.*)/` in the shader source with `#define DEVICE_PIXEL_RATIO 2.0`. Are there occurrences of `/(.*)#define(.*)DEVICE_PIXEL_RATIO(.*)/` in the shader source?